scripts/taux-de-penetration/ajout_agences_milo_sans_conseiller_cej.js

```diff
@@ -0,0 +1,71 @@
+const { Sequelize } = require('sequelize')
+const sequelize = new Sequelize(
+  'postgresql://passemploi:passemploi@localhost:55432/passemploidb',
+  {
+    dialect: 'postgres'
+  }
+)
+
+const referentiel_agences_milo = require('./referentiel_agences_api_milo_copy.json')
+
+async function execute() {
+  const [agencesCej, _metadata] = await sequelize.query(`
+            SELECT *
+            FROM agence
+            WHERE structure = 'MILO'
+             AND id != '9999'
+  `)
+
+  let agencesNonCejReferentiel = await recupererLesAgencesSansConseillersCej(agencesCej)
+  ajouterLesInformationsGeographiques(agencesNonCejReferentiel, agencesCej)
+  /*await persisterLesAgencesSansConseillersCej(agencesNonCejReferentiel)
+  * await persisterLeNomUsuelDansLesAgencesCej*/
+  console.log(agencesNonCejReferentiel)
+}
+
+execute()
+
+async function recupererLesAgencesSansConseillersCej(agencesCej) {
+  return referentiel_agences_milo
+    .map(agenceMilo => {
+      return {
+        id: agenceMilo.codeStructure,
+        nom_officiel: agenceMilo.nomOfficiel,
+        nom_usuel: agenceMilo.nomUsuel
+      }
+    })
+    .filter(
+      agenceMilo => !agencesCej.map(agence => agence.id).includes(agenceMilo.id)
+    )
+}
+
+function ajouterLesInformationsGeographiques(agencesNonCejReferentiel,agencesCej) {
+  for(let agenceNonCej of agencesNonCejReferentiel) {
+    const departement = agenceNonCej.adresse.codePostal.slice(0,2)
+    if(agenceCejTrouvee) {
+
+    } else {
+
+    }
+    agencesCej.find - même département
```

Comment:
pourquoi ? la donnée remontée par leur API n'est pas suffisante ?

Reply:
dans leur retour d'API j'ai :
codeStructure, nomOfficiel, nomUsuel, codePostal
je suppose que tu utilise le codePostal pour matcher le codeDepartement puis tu mets à jour region codeRegion, nomDepartement et timezone yep

Reply:
This is it, comme on a pas de référentiel, je cherche une structure déjà présente en BDD avec le même code